src/plugins/vis_types/timeseries/public/application/components/splits/unsupported_split.js

```diff
@@ -17,14 +15,21 @@ import {
   EuiFormRow,
   EuiCode,
   EuiTitle,
+  useEuiTheme,
 } from '@elastic/eui';
-
 import { FormattedMessage } from '@kbn/i18n-react';
+import { css } from '@emotion/react';
+import { createSelectHandler } from '../lib/create_select_handler';
+import { GroupBySelect } from './group_by_select';
 
 export const SplitUnsupported = (props) => {
   const { onChange, model, uiRestrictions } = props;
   const htmlId = htmlIdGenerator();
   const handleSelectChange = createSelectHandler(onChange);
+  const { euiTheme } = useEuiTheme();
+  const titleStyles = css`
+    margin-top: -${euiTheme.size.xs};
+  `;
```

Comment:
This same style is used three different times in this PR. Can likely be consolidated in the common.styles.ts file.

Reply:
done